packages/dashmate/configs/system/base.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
   },
   core: {
     docker: {
-      image: 'dashpay/dashd:18.0.0.0-rc3',
+      image: 'dashpay/dashd:0.17.0.3',
```

Comment:
Does it work?

Reply:
Since we are shipping a `mainnet.js` config without overriding the value for this image, then yes, this will work on mainnet and other configs based on `base.js` without overriding this value. `testnet.js` now contains an override matching what we want people to be running on testnet.

Reply:
Shouldn't it be more safe and more logical to have a specific version in the mainnet config?

Reply:
OK good point, I made a couple more commits and it is much more simple now